README.md

```diff
@@ -6,6 +6,10 @@ Change facebook voice messages to text messages.
 
 Extension is currently in build.
 
+## Changelog
+
+You can see the most recent changes [here](https://github.com/Zeraye/facebook-speech-recognition/blob/main/CHANGELOG.md).
```

Comment:
You can use a relative path to the changelog so that it works on other branches too. For example, when reading the readme on branch `abc`, the link to the readme will always point to the changelog in the `main` branch - we probably want it to point to the `abc` branch's changelog :smile: 

```suggestion
You can see the most recent changes [here](CHANGELOG.md).
```